doc/source/ray-air/examples/convert_existing_pytorch_code_to_ray_air.ipynb

```diff
@@ -1090,7 +1090,7 @@
    "source": [
     "import ray.data\n",
     "\n",
-    "ds = ray.data.from_items([x.numpy() for x, y in test_data])"
+    "ds = ray.data.from_items([x.numpy() for x, y in test_data], parallelism=8)"
```

Comment:
This and the `real_classes` dataset construction needs hard-coded parallelism in order to ensure that the same number of blocks are created for the downstream `predicted_classes.zip(real_classes)`, which requires each dataset to have the same number of blocks.

Reply:
can you leave this comment inline in the notebook? otherwise someone will randomly remove

Reply:
I removed our use of the `Dataset.zip()` API because of that leaky "same number of blocks" requirement, but I'm leaving `parallelism=8` to demonstrate manually setting the parallelism, and since it results in more sensible downstream parallelism than the 200 default.